oci/archive/oci_src_test.go

```diff
@@ -1,5 +1,26 @@
 package archive
 
-import "github.com/containers/image/v5/internal/private"
+import (
+	"errors"
+	"path/filepath"
+	"testing"
+
+	"github.com/containers/image/v5/internal/private"
+	"github.com/containers/image/v5/types"
+	"github.com/stretchr/testify/assert"
+)
 
 var _ private.ImageSource = (*ociArchiveImageSource)(nil)
+
+func TestNewImageSourceNotFound(t *testing.T) {
+	emptyDir := t.TempDir()
+	sysctx := types.SystemContext{BlobInfoCacheDir: emptyDir}
+	archivePath := filepath.Join(emptyDir, "foo.ociarchive")
+	imgref, err := ParseReference(archivePath)
+	assert.Nil(t, err)
```

Comment:
(`require.NoError(t, err)` would abort the test instead of continuing and crashing.

That’s completely hypothetical here, and it be changed if this ever failed.)